### ISSUE TITLE
Is it really necessary to have always-online DRM for keyboard themes?

### ISSUE THREAD

**Issue by Tiktalik** [ValveSoftware/SteamOS#979]
#### Your system information

* Steam client version: 1675222473
* SteamOS version: 20221228.1
* Opted into Steam client beta?: No 
* Opted into SteamOS beta?: No
* Have you checked for updates in Settings > System?: Yes

#### Please describe your issue in as much detail as possible:

When the Steam Deck loses connection to the internet, the on-screen keyboard theme reverts to the default black one.

The on-screen keyboard is a core part of the operating system, especially in desktop mode, where it's constantly required to interact with software. It feels like an odd choice to have custom themes for it, of all things, require an active internet connection.

Can't the list of themes the user has access to be cached, or only files for keyboard themes the user owns be downloaded, that way the users' OS customization choices can be respected with or without being online?

#### Steps for reproducing this issue:

1. Lose connection to the internet. 
2. On screen keyboard automatically changes to the default theme.

**Comment by TDuffinNTU** [comment]
Agreed. dont think its as insidious as DRM but definitely should be cached
